app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,27 @@
+class PasswordResetsController < ApplicationController
+
+  def show
+    user = User.where(token: params[:id]).first
```

Comment:
You should use `.find_by` to fetch a single record from database instead of `.where(args).first`. In Rails 3, `.where().first` was an OK way, thats why Kevin uses that in videos, but in Rails 4 `.find_by` is the new standard as its more efficient.